modules/lazyload/assets/js/frontend.js

```diff
@@ -8,7 +8,7 @@ document.addEventListener( 'DOMContentLoaded', function() {
 
                 const lazyloadSelector = lazyloadBackground.getAttribute( dataAttribute );
                 if ( lazyloadSelector ) {
-                    lazyloadBackground = lazyloadSelector;
+                    lazyloadBackground = element.querySelector( lazyloadSelector );
```

Comment:
Are we sure all those selectors are inside the element?

Reply:
Well, it should be, because then how it's the background of the element?
Anyway, currently it looks like the only place where the selector is used is inside the element.